<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CacheMapManager</class>
 <widget class="QWidget" name="CacheMapManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="m_btn_add">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cbx_recordList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_ok">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_cancel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="m_label_lng">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>经度</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_label_lat">
      <property name="text">
       <string>纬度</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="m_label_rightDown">
      <property name="text">
       <string>右下:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_lineEdit_rightDownLng"/>
    </item>
    <item>
     <widget class="QLineEdit" name="m_lineEdit_rightDownLat"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="m_label_rightUp">
      <property name="text">
       <string>右上:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_lineEdit_rightUpLng"/>
    </item>
    <item>
     <widget class="QLineEdit" name="m_lineEdit_rightUpLat"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>90</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="m_label_leftUp">
      <property name="text">
       <string>左上:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_lineEdit_leftUpLng"/>
    </item>
    <item>
     <widget class="QLineEdit" name="m_lineEdit_leftUpLat"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="m_label_leftDown">
      <property name="text">
       <string>左下:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_lineEdit_leftDownLng"/>
    </item>
    <item>
     <widget class="QLineEdit" name="m_lineEdit_leftDownLat"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="m_label_zoom">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>层数：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cbx_zooms">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
